sdk/rust/oak/src/lib.rs

```diff
@@ -17,6 +17,8 @@
 //! SDK functionality that provides idiomatic Rust wrappers around the
 //! underlying Oak platform functionality.
 
+#![feature(result_flattening)]
```

Comment:
Interesting, I didn't know about this feature, is it necessary because of some of the new code?

Reply:
Sorry, I was sloppy in my answer! This is needed for the `flatten` in the `Invocation::receive` in `sdk/rust/oak/src/http/mod.rs`.